torch/nn/modules/instancenorm.py

```diff
@@ -59,6 +62,64 @@ def forward(self, input: Tensor) -> Tensor:
             self.training or not self.track_running_stats, self.momentum, self.eps)
 
 
+class _LazyInstanceNorm(LazyModuleMixin, _InstanceNorm):
+
+    weight: UninitializedParameter  # type: ignore[assignment]
+    bias: UninitializedParameter  # type: ignore[assignment]
+
+    cls_to_become = _InstanceNorm  # type: ignore[assignment]
```

Comment:
nit: This isn't required since it's overwritten for the subclasses anyway

Reply:
Done!